### PR TITLE
different smoothing in times

### PR DESCRIPTION
**Pull request by drcandacemakedamoore**
@wbaccinelli  please check my new smoother, it should work better

### CODE REVIEW

**Comment by wbaccinelli** [comment]
@drcandacemakedamoore I tried to apply this to a breath signal (still cut with the tail), but I get the same shape, and the highest point is still the first one

File: resurfemg/helper_functions.py [hunk 1]
```diff
@@ -1215,6 +1215,17 @@ def simple_area_under_curve(
     return area
 
 
+def running_smoother(array):
+    """
+    This is the smoother to use in time calculations
+    """
+    n = len(array) // 10
+    new_list = np.convolve(array, np.ones(n), "valid") / n
```
Comment (wbaccinelli) [inline resurfemg/helper_functions.py]:
```suggestion
    new_list = np.convolve(abs(array), np.ones(n), "valid") / n
```

**Review by wbaccinelli — APPROVED** [review]
NIce, it works!

**Comment by drcandacemakedamoore** [comment]
Good point Walter.

**Pull request merged**